src/components/Content/Content.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Header } from './Header';
+import { TodoList } from './TodoList';
+import { Footer } from './Footer';
+import { Todo } from '../../types/Todo';
+import { TodoStatus } from '../../types/TodoStatus';
+
+type Props = {
+  todos: Todo[];
+  newTodo: React.RefObject<HTMLInputElement>;
+  activeTodos: number;
+  completedTodos: number;
+  todoFilter: TodoStatus;
+  isTodos: boolean;
+  onTodoFilter: (filterStatus: TodoStatus) => void;
+};
+
+export const Content: React.FC<Props> = ({
+  todos,
+  newTodo,
+  activeTodos,
+  completedTodos,
+  isTodos,
```

Comment:
don't pass it as prop, check it here
+ you do a check all todos length when displaying filtered (you should check it here, only filtered todos)

Reply:
If I will not check all todos length here Footer will be hidden on tab Completed (if all todos will have status Active) and there will not be possible to come back to todo list. That's why I use isTodos(todos.length > 0) as a prop.

Reply:
you can pass original todos as a prop and declare everything else here, you use only error in the app.js

Reply:
> you can pass original todos as a prop and declare everything else here, you use only error in the app.js

Thank you for kind reply and recommendations. 😊
I'm absolutely agree with you and will act accordingly! 😉

Reply:
Done ✅